cmd/memberagent/main.go

```diff
@@ -103,7 +102,9 @@ var (
 	workApplierRequeueRateLimiterExponentialBaseForFastBackoff                       = flag.Float64("work-applier-requeue-rate-limiter-exponential-base-for-fast-backoff", 1.2, "If set, the work applier will start to back off fast at this factor after it completes the slow backoff stage, until it reaches the fast backoff delay cap. Its value should be larger than the base value for the slow backoff stage.")
 	workApplierRequeueRateLimiterMaxFastBackoffDelaySeconds                          = flag.Float64("work-applier-requeue-rate-limiter-max-fast-backoff-delay-seconds", 900, "If set, the work applier will not back off longer than this value in seconds when it is in the fast backoff stage.")
 	workApplierRequeueRateLimiterSkipToFastBackoffForAvailableOrDiffReportedWorkObjs = flag.Bool("work-applier-requeue-rate-limiter-skip-to-fast-backoff-for-available-or-diff-reported-work-objs", true, "If set, the rate limiter will skip the slow backoff stage and start fast backoff immediately for work objects that are available or have diff reported.")
-	featureGates                                                                     = featuregates.MemberAgentFeatureGate
+	// Azure property provider feature gates.
+	isAzurePropertyProviderCostCollectionEnabled               = flag.Bool("enable-cost-collection-in-azure-property-provider", true, "If set, the Azure property provider will collect cost information in the member cluster.")
+	isAzurePropertyProviderAvailableResourcesCollectionEnabled = flag.Bool("enable-available-resources-collection-in-azure-property-provider", true, "If set, the Azure property provider will track available resources in the member cluster.")
```

Comment:
nit: anyway to make the flat name a bit short?

Reply:
Done 👌